### PR TITLE
Use new signal_t struct.

### PR DESCRIPTION
**Pull request by ungureanuvladvictor**
Signed-off-by: Vlad Ungureanu <vvu@vdev.ro>

### CODE REVIEW

**Comment by ungureanuvladvictor** [comment]
Any idea how we should solve the build issue? Which names should we use where...

\cc: @fnoble @gsmcmullin @mfine

File: src/acq.c [hunk 1]
```diff
@@ -44,11 +44,11 @@ void acq_set_prn(u8 prn)
  * \param cp  Code phase of best point.
  * \param cf  Carrier frequency of best point.
  */
-void acq_send_result(u8 prn, float snr, float cp, float cf)
+void acq_send_result(u16 prn, float snr, float cp, float cf)
 {
   msg_acq_result_t acq_result_msg;
 
-  acq_result_msg.sid = prn; /* TODO prn -> sid */
+  acq_result_msg.sid.prn = prn; /* TODO prn -> sid */
```
Comment (mfine) [inline src/acq.c]: Clear the TODO? Or is it still helpful to leave it around to use the other information in the structure?

Comment (ungureanuvladvictor) [inline src/acq.c]: We still have some places where constellation/band is not used. Will remove later when I come back to this.

**Comment by gsmcmullin** [comment]
Replaced by #561

**Pull request closed without merging**